sdk/search/azure-search-index/azure/search/index/_generated/_version.py

```diff
@@ -0,0 +1,7 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator (autorest: 3.0.6185, generator: {generator})
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+VERSION = "None"
```

Comment:
I know this is in the generated code, but I'm used to seeing the service version here. Not sure how you're setting the user-agent yet, but this might result in `User-Agent: azsdk-python-searchindexclient/None`